tests/integration/component.go

```diff
@@ -411,4 +411,32 @@ func componentTests(args ...string) {
 			helper.CmdShouldPass("odo", "delete", cmpName, "--app", appName, "--project", project, "-f")
 		})
 	})
+
+	Context("successive odo push to check for existence of environment variables", func() {
+		JustBeforeEach(func() {
+			project = helper.CreateRandProject()
+			context = helper.CreateNewContext()
+			originalDir = helper.Getwd()
+		})
+
+		JustAfterEach(func() {
+			helper.DeleteProject(project)
+			helper.DeleteDir(context)
+			os.RemoveAll(context)
+		})
+
+		It("should create a component with environment variables and the env vars should persist multiple odo push", func() {
+			helper.CmdShouldPass("git", "clone", "https://github.com/sclorg/nodejs-ex", context)
```

Comment:
Please use `CopyExample` function. You can see it in other tests. Cloning form git will slow tests down, and introduce possible flakes due to networking issues

Reply:
Ack, thanks.